<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laboratorio Prototipado Unal-La Paz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 180, 59);
color: rgb(255, 255, 255);
border-radius:8px</string>
    </property>
    <property name="text">
     <string>Laboratorio De Prototipado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Logotipo.unal.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="btn_import_Excel_a_db"/>
    <addaction name="btn_import_db"/>
   </widget>
   <widget class="QMenu" name="menuRegistro">
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="btn_Reg_Usuario"/>
    <addaction name="btn_Reg_Equipo"/>
    <addaction name="btn_Reg_Insumos"/>
   </widget>
   <widget class="QMenu" name="menuPrestamo">
    <property name="title">
     <string>Prestamo</string>
    </property>
    <addaction name="btn_Prest_Insumos"/>
    <addaction name="btn_Prest_Equipos"/>
   </widget>
   <widget class="QMenu" name="menuDevolucion">
    <property name="title">
     <string>Devolucion</string>
    </property>
    <addaction name="btn_Dev_Equipos"/>
    <addaction name="btn_Dev_Insumos"/>
   </widget>
   <addaction name="menArchivo"/>
   <addaction name="menuRegistro"/>
   <addaction name="menuPrestamo"/>
   <addaction name="menuDevolucion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btn_import_Excel_a_db">
   <property name="text">
    <string>Importar Excel a db</string>
   </property>
  </action>
  <action name="btn_import_db">
   <property name="text">
    <string>importar db</string>
   </property>
  </action>
  <action name="btn_Reg_Usuario">
   <property name="text">
    <string>Usuario</string>
   </property>
  </action>
  <action name="btn_Reg_Equipo">
   <property name="text">
    <string>Equipos</string>
   </property>
  </action>
  <action name="btn_Reg_Insumos">
   <property name="text">
    <string>Insumos</string>
   </property>
  </action>
  <action name="btn_Prest_Insumos">
   <property name="text">
    <string>Insumos</string>
   </property>
  </action>
  <action name="btn_Prest_Equipos">
   <property name="text">
    <string>Equipos</string>
   </property>
  </action>
  <action name="btn_Dev_Equipos">
   <property name="text">
    <string>Equipos</string>
   </property>
  </action>
  <action name="btn_Dev_Insumos">
   <property name="text">
    <string>Insumos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
